uniffi_bindgen/src/bindings/mod.rs

```diff
@@ -130,21 +130,35 @@ pub fn compile_bindings<P>(
     config: &Config,
     ci: &ComponentInterface,
     out_dir: P,
+    extra_files: &[PathBuf],
     language: TargetLanguage,
 ) -> Result<()>
 where
     P: AsRef<Path>,
 {
     let out_dir = out_dir.as_ref();
     match language {
-        TargetLanguage::Kotlin => kotlin::compile_bindings(&config.kotlin, ci, out_dir)?,
-        TargetLanguage::Swift => swift::compile_bindings(&config.swift, ci, out_dir)?,
-        TargetLanguage::Python => (),
+        TargetLanguage::Kotlin => {
+            kotlin::compile_bindings(&config.kotlin, ci, extra_files, out_dir)?
+        }
+        TargetLanguage::Swift => swift::compile_bindings(&config.swift, ci, extra_files, out_dir)?,
+        TargetLanguage::Python => copy_extra_files(extra_files, out_dir)?,
```

Comment:
A comment here would be good to explain why Python is `copying`, Ruby is doing nothing, and Swift and Kotlin are doing approximately the same.

It looked odd, until I looked at the context and thought about it. Don't make me think, gosh dern it. :)